shell/platform/darwin/ios/framework/Source/FlutterPlatformPlugin.mm

```diff
@@ -41,15 +41,15 @@
 using namespace flutter;
 
 static void SetStatusBarHiddenForSharedApplication(BOOL hidden) {
-#if APPLICATION_EXTENSION_API_ONLY
+#if not APPLICATION_EXTENSION_API_ONLY
```

Comment:
I've never seen `not` in a preprocessor directive. Is this a non-standard extension?

Reply:
The macro `not` as alternative representation of `!` in cpp.

Related links are as follows
- [Alternative operator spellings - macro](https://cplusplus.com/reference/ciso646/)
- [‘#if’ directive](https://gcc.gnu.org/onlinedocs/cpp/If.html)